frontend/pages/SoftwarePage/SoftwareVulnerabilities/SoftwareVulnerabilitiesTable/_styles.scss

```diff
@@ -0,0 +1,40 @@
+.software-vulnerabilities-table {
+  &__count {
+    display: flex;
+    gap: 12px;
+    align-items: center;
+  }
+
+  .hosts-cell__wrapper {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .vulnerabilities-hosts-link {
```

Comment:
this and the below class are encapsulated in `ViewAllHostsLink` now so I think we can remove these, if we haven't done so already (I know there have been updates since this PR)